.github/workflows/component_features.yml

```diff
@@ -0,0 +1,69 @@
+name: Component Features - Linux
+
+on:
+  workflow_call:
```

Comment:
I did consider that. The reasons I organized it this way and IMO the advantages are:
 - It is easier to know what is in the merge queue, because we aren't allowed to have workflows in directories, they all have to be flat in the .workflows dir, now we can see from the single master_merge_queue.yml file, what all is in the merge queue and that file itself is (mostly, with exception of the int tests), not complicated.
 - I think it is easier to click on the one workflow run 'Master Merge Queue Suite' and all the jobs there, rather than have to open a new tab to view each of them.
 
 I realize that is kind of opinionated so I'm def open to differing thoughts.

Reply:
Gotcha, yeah, those seem like nice benefits so I'm happy to go with it. I was just curious if you considered both ways.